tasks/go_test.py

```diff
@@ -663,7 +667,7 @@ def send_unit_tests_stats(_, job_name):
             timestamp,
             n_test_fast,
             tags=[
-                "experimentation:fast-tests",
+                "experimentation:fast-tests-v2",
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
You could add extra tags here to identify your version

Reply:
For this one I can yes! I will update it.
However I think that I must rename the job because the data about duration are collected directly by CI Visibility and I do not think I can give different tags to a job with the same name to CI Visibility